frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -7,12 +7,19 @@ import {
 
 type LogsTabStatusProps = {
   error?: Error;
+  isLogsNotAvailable?: boolean;
```

Comment:
Prefer a positive boolean prop:
```suggestion
  isLogsAvailable?: boolean;
```